icu4j/tools/build/src/main/java/com/ibm/icu/dev/tool/errorprone/ErrorProneReport.java

```diff
@@ -266,4 +268,107 @@ private static void outSummary(HtmlUtils hu, Map<String, List<ErrorProneEntry>>
             hu.closeTag("p").nl();
         }
     }
+
+    private static void genReportTsv(Map<String, List<ErrorProneEntry>> errors,
+            String outDir) throws IOException {
+        Path outFileName = Paths.get(outDir, TSV_REPORT_FILE);
+        System.out.println("Report generated: " + outFileName);
+        try (PrintStream wrt = new PrintStream(outFileName.toString(), StandardCharsets.UTF_8)) {
+            wrt.println("Issue type"
+                    + "\tCount"
+                    + "\tSeverity"
+                    + "\tURL"
+            );
```

Comment:
optional: you could of course just write one string "Issue type\tCount\t..."

Reply:
The reason for that was to make it more readable and easy to keep in sync with the values, a bit below:

```java
wrt.println("Issue type"
        + "\tCount"
        + "\tSeverity"
        + "\tURL"
);
...
wrt.println(errorType
        + "\t" + errorsOfType.size()
        + "\t" + errorSeverity
        + "\t" + firstEntry.url
);
```

So if you add another column it is easier.